docker-compose.yml

```
@@ -290,7 +296,7 @@ services:
       - RECOMMENDATION_SERVICE_ADDR
       - SHIPPING_SERVICE_ADDR
       - OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
-      - OTEL_RESOURCE_ATTRIBUTES=service.name=frontend
+      - OTEL_RESOURCE_ATTRIBUTES=${OTEL_RESOURCE_ATTRIBUTES},service.name=frontend
```

```suggestion
      - OTEL_RESOURCE_ATTRIBUTES=${OTEL_RESOURCE_ATTRIBUTES}
```

I think `service-name` was a leftover here.
We have `OTEL_SERVICE_NAME=frontend` defined below, so we are good to remove it from the `OTEL_RESOURCE_ATTRIBUTES`.